src/AppBundle/Resources/translations/messages.en.yml

```diff
@@ -6,6 +6,9 @@ navbar.myAccount: My Account
 base.footer.contactUs: Contact us
 base.footer.help: Help
 base.footer.legal: Legal notice
+base.footer.suggest: Suggest a restaurant
```

Comment:
`base.footer.suggest` & `form.suggest.title` are the same sentence, [DRY](https://fr.wikipedia.org/wiki/Ne_vous_r%C3%A9p%C3%A9tez_pas)